api.js

```diff
@@ -69,8 +69,8 @@ function getLocationFromVF(mmsi, cb) {
       const latitude = splitted[0].indexOf('N') === -1 ? parseFloat(splitted[0]) * -1 : parseFloat(splitted[0]);
       const longitude = splitted[1].indexOf('E') === -1 ? parseFloat(splitted[1]) * -1 : parseFloat(splitted[1]);
 
-      const timestamp = new Date($('.vfix-top:nth-of-type(1) .tparams tr:nth-of-type(11) .v3').text()).toString();
-      const unixtime = new Date($('.vfix-top:nth-of-type(1) .tparams tr:nth-of-type(11) .v3').text()).getTime()/1000;
+      const timestamp = new Date($('#lastrep').attr('data-title')).toString();
+      const unixtime = new Date(timestamp).getTime()/1000;
```

Comment:
No need to perform another query here - we already have the timestamp!

Reply:
That certainly seems to fix the Vesselfinder scraping... but not the marine traffic (yet) this returns only could not parse date.

The date seems to be here:
![image](https://user-images.githubusercontent.com/59790175/73526071-b2e8fb00-4410-11ea-8ee8-0c8a3664c66f.png)

What would need to be changed in the API code to pull the date/time from MT as well ?